ndcube/extra_coords/table_coord.py

```diff
@@ -16,6 +16,14 @@
 __all__ = ['TimeTableCoordinate', 'SkyCoordTableCoordinate', 'QuantityTableCoordinate']
 
 
+@models.custom_model
+def length1_lookup_table(x, lookup_table, fill_value=np.nan):
+    if x.to_value(u.pix) == 0:
```

Comment:
This is too narrow. A pixel has a width!

Reply:
So it should be this?
```suggestion
    x_is_scalar = False
    if x.isscalar():
        x_is_scalar = True
        x.reshape((1,))
    output = np.full(x.shape, fill_value)
    x_value = x.to_value(u.pix)
    output[np.logical_or(x_value >= -0.5, x_value < 0.5)] = lookup_table[0].value
    output *= lookup_table[0].unit
    if x_is_scalar:
        return output[0]
    return output
```

Reply:
I think that would work, would need some tests just for the model I think.